recipes/eabase/all/conanfile.py

```diff
@@ -0,0 +1,44 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class EABaseConan(ConanFile):
+    name = "eabase"
+    description = "EABase is a small set of header files that define platform-independent data types and platform feature macros. "
+    topics = ("conan", "eabase", "config",)
+    license = "BSD 3-Clause"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```

Please make sure that you are using valid SPDX identifier https://spdx.org/licenses/